web/modules/custom/dpl_loans/src/Plugin/Block/LoanListBlock.php

```diff
@@ -98,7 +87,7 @@ public function build() {
       "page-size-mobile" => $loanListSettings->get('page_size_mobile') ?? DplLoansSettings::PAGE_SIZE_MOBILE,
 
       // Config.
-      "threshold-config" => $this->getThresholdConfig(),
+      "expiration-warning-days-before" => $generalSettings->get('expiration_warning_days_before_config') ?? GeneralSettingsForm::EXPIRATION_WARNING_DAYS_BEFORE_CONFIG,
```

Comment:
You miss the `-config` suffix right?
Shouldn't it be: `expiration-warning-days-before-config`?

Reply:
You are right. This should have been fixed now. :)